src/Microsoft.AspNetCore.Http/HeaderDictionary.cs

```diff
@@ -97,6 +98,34 @@ public HeaderDictionary(int capacity)
             set { this[key] = value; }
         }
 
+        public long? ContentLength
+        {
+            get
```

Comment:
I thought about it but the implementation is actually different, their internal stores work a little differently.